docs/content/guide/index.mdx

```diff
@@ -83,14 +83,23 @@ catalog.jinja_env.tests.update({ ... })
 catalog.jinja_env.extensions.extend([ ... ])
 ```
 
-You can also reuse an existing Jinja Environment, for example, the one from **Flask**:
+You can also reuse an existing Jinja Environment, for example:
+ **Flask**:
 
 ```python
 app = Flask(__name__)
 
 # Here we add the flask Jinja globals, filters, etc. like `url_for()`
 catalog = jinjax.Catalog(jinja_env=app.jinja_env)
 
+```
+ **Django**:
+(configure jinja in setting.py and [jinja_env.py](https://docs.djangoproject.com/en/5.0/topics/templates/#django.template.backends.jinja2.Jinja2))
+```python
+env.add_extension(JinjaX)
+catalog = Catalog(jinja_env=env)
+catalog.add_folder("components")
+
```

Comment:
True, it would be good have a separate  "components" folder for shared components and also have "components" subfolder at django-app level. so, it should be:

```
import jinjax
from jinja2.loaders import FileSystemLoader


def environment(loader: FileSystemLoader, **options):
    env = Environment(loader=loader, **options)

    ...

    env.add_extension(jinjax.JinjaX)
    catalog = jinjax.Catalog(jinja_env=env)

    catalog.add_folder("components")
    for dir in loader.searchpath:
        catalog.add_folder(os.path.join(dir, "components"))

    return env
```

Reply:
I'm not sure the root `catalog.add_folder("components")` is really necessary since `loader.searchpath` will contain all `DIRS` (and each app's `jinja2` folder if `APP_DIRS: True`).

To clarify the behaviour here, the app component paths end up as `<app>/jinja2/components`. Personally I like that nesting...